source/code/source/playfab/PlayFabCallRequestContainer.cpp

```diff
@@ -80,6 +80,9 @@ namespace PlayFab
         bool isValid = true;
         if (m_settings->titleId.empty())
         {
+            // Option B
+            //throw new PlayFabException(PlayFabExceptionCode::TitleNotSet, "PlayFabSettings::staticSettings->titleId has not been set properly. It must not be empty.");
```

Comment:
Option B obviously also involves deleting essentially all the rest of the code in this method.
I do like it though, because it is a more direct and instantaneous feedback, at the point of failure.
The current mechanism returns the error as a callback, which is disconnected from the point of failure.